src/views/error.ejs

```diff
@@ -1,17 +1,26 @@
-<div class="card-container">
-    <div class="fr-alert fr-alert--error">
-        <p class="fr-alert__title">Une erreur est survenue.</p>
-        <pre class="error-code">
-            <b>Erreur <%= error_code %></b>
-            <%= error_message %>
-        </pre>
-        <p>
-            Merci de réessayer ultérieurement.
-        </p>
-        <p>
-            Vous pouvez également nous signaler cette erreur par mail à
-            <a href="mailto:contact@moncomptepro.beta.gouv.fr?subject=Erreur%20sur%20MonComptePro">contact@moncomptepro.beta.gouv.fr</a>.
-        </p>
+<div class="fr-grid-row fr-grid-row--center fr-grid-row--middle mcp-full-height">
+    <div class="fr-col-lg-5 fr-col-offset-lg-1--right fr-col-md-7 fr-col-sm-8 fr-pt-5w mcp-full-height">
```

Comment:
these two lines are in every template (except where fr-mb-6w is missing, which seems like an omission). I think we could put these line in the layout.ejs :

```ejs
<div class="fr-grid-row fr-grid-row--center fr-grid-row--middle mcp-full-height">
    <div class="fr-col-lg-5 fr-col-offset-lg-1--right fr-col-md-7 fr-col-sm-8 fr-pt-5w mcp-full-height">
        <%- body; %>
    </div>
    <div
        class="fr-col-lg-6 fr-py-5w mcp-full-height side-media"
    >
        <div class="fr-grid-row fr-grid-row--center">
            <img src="/dist/<%= illu %>" class="fr-responsive-img" alt="" />
        </div>
    </div>
</div>
```